NodeServer/nodeserver.js

```diff
@@ -62,6 +64,7 @@ var consumer = io.of( '/consumer' );
 consumer.on( 'connection', function( socket ) {
   // Send an update to the newly connected consumer socket
   socket.emit( 'stats-updated', stats );
+  socket.emit( 'create', stats );
```

Comment:
currently I have only added "create" event on the server side. If you want to perform and support the other operations as well - you should add "socket.emit('destroy', args);" and "socket.emit('update', args);"